### ISSUE TITLE
Regarding adding an example on Gradient Centralization

### ISSUE THREAD

**Issue by Rishit-dagli** [keras-team/keras-io#418]
I plan to propose an example demonstrating an implementation of the paper [Gradient Centralization: A New Optimization Technique for Deep Neural Networks](https://arxiv.org/abs/2004.01461) which can both speed up the training process and improve the final generalization performance of DNNs especially for general image classification, fine-grained image classification, detection and segmentation and Person ReID.

I have also brought this to TensorFlow as a [ready to use Python package](https://github.com/Rishit-dagli/Gradient-Centralization-TensorFlow) to use with Keras. Here is an example notebook from this repository [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Rishit-dagli/Gradient-Centralization-TensorFlow/blob/main/examples/gctf_horses_v_humans.ipynb) and I plan to propose a similar one with any changes if required.

**Comment by Rishit-dagli** [comment]
@fchollet I was wondering if you had the chance to consider this? I do understand if you might have been swamped.

**Comment by sayannath** [comment]
Hi @Rishit-dagli you can directly make a PR! This example is great and it gonna help many people.

**Comment by Rishit-dagli** [comment]
> Hi @Rishit-dagli you can directly make a PR! This example is great and it gonna help many people.

Sure thing, I could directly make one

**Comment by sayakpaul** [comment]
@Rishit-dagli can we close this issue?

**Comment by Rishit-dagli** [comment]
> @Rishit-dagli can we close this issue?

@sayakpaul I have linked #497 with this issue so it would close on merging the PR, should I close this issue now?

**Comment by sayakpaul** [comment]
No need then. Thanks.
